plugins/modules/update_image.py

```diff
@@ -11,25 +11,41 @@
 DOCUMENTATION = """
 ---
 module: update_image
-short_description: Update a new RHEL for Edge Image on console.redhat.com
+short_description: updates latest image to a new image version
 description:
-  - This module will build a RHEL for Edge Image on console.redhat.com
+  - This module will build the latest version for a RHEL for Edge Image on console.redhat.com
 version_added: "0.1.0"
 options:
-  id:
+  name:
     description:
-      - ID of Image that will be updated based on this new Image build request
+      - Name of image that will be updated
     required: true
-    type: int
+    type: str
   packages:
     description:
-      - Aditional packages to add for this Image update build
+      - Aditional packages to add for the new image
     required: false
     type: list
     elements: str
+  output_type:
+    description:
+      - Type of image that will be built.
+    required: false
+    type: str
+    default: commit
+  username:
+    description:
+      - username that will be used for the new image
+    required: false
+    type: str
+  sshkey:
+    description:
+      - sshkey that will be used for the new image
+    required: false
+    type: str
 
 author:
-  - Adam Miller (@maxamillion)
```

Comment:
Just a note: Since you rewrote this whole-cloth, this is fine but in a more "normal" iterative changeset you just keep adding to it because as code iterates the previous author doesn't lose their authorship claims.